app/controllers/sessions_controller.rb

```diff
@@ -6,25 +6,25 @@ def new
 
   def create
     auth = request.env["omniauth.auth"]
-    user = User.where(:provider => auth['provider'],
-                      :provider_id => auth['uid'].to_s).first || User.create_from_omniauth!(auth)
+    user = User.where(provider: auth['provider'],
+                      provider_id: auth['uid'].to_s).first || User.create_from_omniauth!(auth)
     user.update_credentials(auth)
     sign_in user
     if user.email.blank?
-      redirect_to edit_user_path(user), :alert => "Please enter your email address."
+      redirect_to edit_user_path(user), alert: "Please enter your email address."
     else
-      redirect_to root_url, :notice => 'Signed in!'
+      redirect_to root_url, notice: 'Signed in!'
     end
   end
 
   def destroy
     sign_out
-    redirect_to root_url, :notice => 'Signed out!'
+    redirect_to root_url, notice: 'Signed out!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.